app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Qualification - (non-UK) country",
+  items: {
+    question: "In which country did you gain this qualification?",
+    required: "Yes (if non-UK qualification)",
+    api: {
+      param: "qualifications.gcses[].awarding_body",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "needs to look for evidence that a qualification is of a standard equivalent to GCSE grade 4, not only in terms of its level, but also in terms of its breadth. Details of qualification can help determine if equivalent to GCSE"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: ["If we are able to use structured data, do we then need to capture information about the country of the awarding body?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Qualification - (non-UK) equivalency",
+  items: {
+    question: "Do you know what UK qualification your maths qualification is equivalent to?",
+    required: "Yes (if non-UK qualification)",
+    api: {
+      param: "qualifications.gcses[].equivalency_details",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "needs to provide evidence that their international qualification demonstrates a similar level and breadth as the equivalent UK qualification"
+    }, {
+      who: ["candidate"],
+      what: "may already have a statement of comparability from NARIC"
+    }, {
+      who: ["provider"],
+      what: "needs to look for evidence that a qualification is of a standard equivalent to GCSE grade 4, not only in terms of its level, but also in terms of its breadth. Details of qualification can help determine if equivalent to GCSE"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
+    notes: ["Candidates can request a statement of comparability from NARIC, or speak to a TTA who can certify equivalence"]
+  }
+}) }}
+
+## English GCSE or equivalent
+
+Candidates can update their answers before (and 5 working days after) submitting their application.
+
+Providers also need further evidence of a breadth of achievement in English where applicants have achieved a GCSE grade 4 or above *in English literature only*. We don’t currently make this distinction in the service.
```

Comment:
I thought it was English language? Interesting that the ITT criteria don't directly reference this.

Reply:
Yeah, guidance could be clearer and explicitly state English language. It does add further detail in the guidance:

> Providers should look for further evidence of a breadth of achievement in English where applicants have achieved a GCSE grade 4 or above in English literature only.